signer/pkcs11/pkcs11.go

```diff
@@ -18,6 +18,7 @@ import (
 // #11 key.
 type Config struct {
 	Module string
+	Slot   string
```

Comment:
So long as we are breaking the API, let's take the opportunity to give more descriptive names to these fields. Suggestions: SlotDescription, TokenLabel, and Label -> PrivateKeyLabel.